tests/login/login.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from '@playwright/test';
+import { caritasRework } from '../config';
+import { ensureLanguage } from '../utils';
+
+test('Login as an advice seeker', async ({ page }) => {
+	const username = process.env.TEST_USERNAME;
+	const password = process.env.TEST_PASSWORD;
+	await page.goto(`${caritasRework.dev}`);
+	ensureLanguage(page);
```

Comment:
We can start with this, but generally I would recommend having the tests working no matter what language is used.